### PR TITLE
CAL-17 boilers type crud

### PR DESCRIPTION
**Pull request by cocolobos**

### CODE REVIEW

**Review by sbmedina — COMMENTED** [review]
Please delete all comments and outdated files.

File: app/controllers/boilersTypes.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
+const db = require("../models");
+const BoilerType = db.boilersTypes;
+exports.create = (req,res) => {
+  //validate request
```
Comment (lupoalejandro) [inline app/controllers/boilersTypes.js:4]: Delete the comment

File: app/controllers/boilersTypes.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    .save(boilerType)
+    .then(data => {
+      res.send(data);
+    })
+    .catch(err => {
+      res.status(500).send({
+        message: "Some error ocurred while creating new Boiler Type"
+      });
+    });
+};
```
Comment (lupoalejandro) [inline app/controllers/boilersTypes.js:25]: add white space between the functions

File: app/routes/boilers-types.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const boilersTypes = require ("../controllers/boilersTypes");
+var router = require ("express").Router();
```
Comment (lupoalejandro) [inline app/routes/boilers-types.js:2]: add space here

File: app/routes/boilers-types.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
...
+router.get('/',boilersTypes.findAll);
+router.get('/:id',boilersTypes.findOne);
+router.post('/',boilersTypes.create);
+router.put('/:id',boilersTypes.update);
+router.delete('/:id',boilersTypes.delete);
+module.exports = router;
```
Comment (lupoalejandro) [inline app/routes/boilers-types.js:8]: add space here

File: index.js [hunk 5]
```diff
@@ -5,10 +5,9 @@ const router = require('./app/routes');
 const bodyParser = require("body-parser");
 
 app.use(bodyParser.json());
-
 app.use(bodyParser.urlencoded({ extended: true}));
 
-const PORT = process.env.PORT || 5000;
+const PORT = process.env.PORT || 3000;
```
Comment (lupoalejandro) [inline index.js:10]: the alternative port is 5000, why did you change it?
